cves/kernel/CVE-2020-25284.yml

```diff
@@ -468,7 +462,13 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    The mistake made in this vulnerability was a lapse. The developer(s)
+    who wrote the additional code did not remember to include a check for if 
+    the user has the appropriate permissions to map or unmap virtual memory 
+    blocks. The fix to the issue was very simple, the developer only had to 
+    add a conditional statement to quit the function call if the user calling 
+    it lacked sufficient privileges.
```

Comment:
Any lessons to be learned from this?

Reply:
I'll add a blurb about unit testing, as I think in this case it would have caused the vulnerability to be caught before it went live.